food-truck-frontend/src/components/map/LocationLoaderComponent.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from "react";
+import { loadUserDefaultLocation } from "../../util/position";
+
+/**
+ * Component for loading the user's current position, if it isn't in localstorage.
+ */
+const LocationLoaderComponent = () => {
+    useEffect(loadUserDefaultLocation);
```

Comment:
you might want to make this `useEffect(loadUserDefaultLocation, [])` to prevent spamming but its fine since it shouldn't spam the DB